crates/workspace/src/workspace.rs

```diff
@@ -1146,6 +1147,7 @@ pub struct Workspace {
     _items_serializer: Task<Result<()>>,
     session_id: Option<String>,
     scheduled_tasks: Vec<Task<()>>,
+    pub which_key: Option<AnyEntity>,
```

Comment:
I'm not 100% sure if this is the correct approach to preventing `WhichKey` from being dropped.